docs/source/training/socialisation/faq.rst

```diff
@@ -74,8 +74,8 @@ QGIS and |project_name|.
 
     - Yes you can, but you need to enable on the fly projection.
     - In QGIS, activate OTF CRS Transformation
-       - Project ? Project Properties ? CRS ?
-       - checkbox in the Enable �on the fly� CRS transformation.
+       - Project > Project Properties > CRS
+       - checkbox in the Enable 'on the fly' CRS transformation.
```

Comment:
Tick the :guilabel:`Enable on the fly transformation` checkbox 

See http://sphinx-doc.org/markup/inline.html#role-guilabel